ghcup.cabal

```diff
@@ -354,9 +406,8 @@ executable ghcup
     other-modules: BrickMain
```

Comment:
once you remove the app/ghcup/BrickMain.hs, Remove this line also

Reply:
Yes, also, the executable can drop pedencies on vty/brick because now those are in the ghcup-tui library.